### PR TITLE
fix: find action by id

### PR DESCRIPTION
**Pull request by machi1990**
##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

File: platform/client-admin/src/pages/ViewActionPage.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { IonLoading, IonContent, IonList, IonCard, IonItemGroup, IonItemDivider
 import recipient from '../forms/recipient';
 
 export const ViewActionPage: React.FC<RouteComponentProps<IUpdateMatchParams>> = ({ match }) => {
-  const { data, loading, error } = useFindVolunteerActionsQuery({ fetchPolicy: "cache-first", variables: { fields: {}, limit: 50 } });
+  const { data, loading, error } = useFindVolunteerActionsQuery({ fetchPolicy: "cache-first", variables: { fields: {id: match.params.id}, limit: 1 } });
```
Comment (machi1990) [inline platform/client-admin/src/pages/ViewActionPage.tsx:13]: should the fetch policy be `network-first`? @wtrocki

Comment (machi1990) [reply platform/client-admin/src/pages/ViewActionPage.tsx:13]: That's what I have for the soon to be edit volunteer page

Comment (wtrocki) [reply platform/client-admin/src/pages/ViewActionPage.tsx:13]: Cache should contain entire result so no need. That is actually the power of graphql cache

Comment (machi1990) [reply platform/client-admin/src/pages/ViewActionPage.tsx:13]: Okay. I thought we probably would want to have an up to date infos here. 
I'll look into the different policies we have for my own culture.

**Pull request merged**